web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -947,7 +985,20 @@ private Object validateStudyEvent(StudyEventDataBean studyEventDataBean, StudySu
     }
 
 
-    public ErrorObj validateStartAndEndDateAndOrder(StudyEventDataBean studyEventDataBean) {
+    private boolean isEventCrfCompleted(EventCrf eventCrf) {
+    	if(eventCrf.getStatusId() == EventCRFStatus.INITIAL_DATA_ENTRY_COMPLETE.getCode()) {
+    		return true;
+    	}else if(eventCrf.getStatusId() == EventCRFStatus.DOUBLE_DATA_ENTRY_COMPLETE.getCode()) {
+    		return true;
```

Comment:
this status DOUBLE_DATA_ENTRY_COMPLETE is not valid anymore, no need to use it

Reply:
removed